mriqc/instrumentation/resources.py

```diff
@@ -170,6 +191,9 @@ def run(self, *args, **kwargs):
         if self._exclude is True:
             self._exclude = (psutil.Process().pid,)
 
+        # Ensure done is not marked set
+        self._done.clear()
+
```

Comment:
The rebase is not well done. This should not be marked as a change

Reply:
Yes, sorry I fixed it